common/properties/coverageLevels.txt

```diff
@@ -152,6 +152,7 @@ tn ;	basic ;	Tswana
 to ;	basic ;	Tongan
 tr ;	modern ;	Turkish
 tt ;	moderate ;	Tatar
+ug ;	basic ;	Uyghur
```

Comment:
This will be overridden as soon as common/properties/coverageLevels.txt is regenerated. But harmless, so approving.

Reply:
Does that happen automatically or does that happen if a contributor runs the script? After this commit it should be always covered.

I figured this is a good test to prove my commit works too ;)